File: app/src/main/java/io/plan8/backoffice/vm/EditTaskActivityVM.java
package io.plan8.backoffice.vm;

import android.os.Bundle;
import android.view.View;
import android.widget.Toast;

import io.plan8.backoffice.activity.EditTaskActivity;

/**
 * Created by chokwanghwan on 2017. 11. 28..
 */

public class EditTaskActivityVM extends ActivityVM {
    private EditTaskActivity editTaskActivity;
    public EditTaskActivityVM(EditTaskActivity editTaskActivity, Bundle savedInstanceState) {
        super(editTaskActivity, savedInstanceState);
        this.editTaskActivity = editTaskActivity;
    }

    public void finishEditTask(View view) {
        editTaskActivity.onBackPressed();
    }

    public void completeEditTask(View view) {
        Toast.makeText(getActivity().getApplicationContext(), "완료", Toast.LENGTH_SHORT).show();
    }
}

File: app/src/main/java/io/plan8/backoffice/vm/EditTaskActivityVM.kt
package io.plan8.backoffice.vm

import android.os.Bundle
import android.view.View
import android.widget.Toast
import io.plan8.backoffice.activity.EditTaskActivity

/**
 * Created by SSozi on 2017. 11. 16..
 */
class EditTaskActivityVM(var editTaskActivity: EditTaskActivity, savedInstanceState: Bundle?) : ActivityVM(editTaskActivity, savedInstanceState) {
    fun finishEditTask(view: View){
        editTaskActivity.onBackPressed()
    }

    fun completeEditTask(view: View){
        Toast.makeText(editTaskActivity, "완료", Toast.LENGTH_SHORT).show()
    }
}